.github/workflows/release.yaml

```diff
@@ -83,25 +83,24 @@ jobs:
           go-version: ${{ env.go-version }}
 
       - name: Generate release notes
+        id: release_notes
         run: |
           # Create the release notes for the Github release
           git config user.name rhobs-release-bot
           git config user.email release-bot@monitoring.rhobs.io
-          npx standard-version -i RELEASE_BODY.md --release-as $(cat VERSION) --skip.commit
+          npx standard-version -i RELEASE_BODY.md  \
+            --release-as $(cat VERSION) \
+            --skip.commit
```

Comment:
Note: unlike old release workflow, this one does not push the tag and only creates a  draft release without a git tag. The git tag gets created  only after the commit that updates index.yaml is added (after running `olm-publish` workflow) and pushed